replications/internal/store.go

```diff
@@ -155,6 +171,9 @@ func (s *Store) UpdateReplication(ctx context.Context, id platform.ID, request i
 	if request.RemoteBucketID != nil {
 		updates["remote_bucket_id"] = *request.RemoteBucketID
 	}
+	if request.RemoteBucketName != nil {
+		updates["remote_bucket_name"] = *request.RemoteBucketName
```

Comment:
hmm - I wonder if we should have also updated the `remote_bucket_id` to nil here, to avoid getting both filled.

not blocking, can be fixed post-release as a follow up if desired.